common/lib/xmodule/xmodule/course_module.py

```diff
@@ -855,6 +855,14 @@ class CourseFields(object):
         scope=Scope.settings
     )
 
+    minimum_grade_credit = Float(
```

Comment:
@cahrens we have already moved the enabling course for credit in openedx. but for minimum grade we don't need any checks for global staff (vs. course staff). 

For help message i will talk to @wedaly.